ganalytics.php

```diff
@@ -268,9 +279,10 @@ public function onAssetsInitialized()
 
         // Parameters
         $scriptName = $this->config->get('plugins.ganalytics.debugStatus', false) ? 'analytics_debug' : 'analytics';
-        $objectName = trim($this->config->get('plugins.ganalytics.objectName', 'ga'));
+        $objectName = trim($this->config->get('plugins.ganalytics.objectName', 'gtag'));
         $async      = $this->config->get('plugins.ganalytics.async', false);
         $position   = trim($this->config->get('plugins.ganalytics.position', 'head'));
+        $isNewMode  = $this->config->get('plugins.ganalytics.ga4', false);
```

Comment:
**Chore:** Same here with `new`. If method is named `getGa4...`, then why here not `isGa4Mode`?